### PR TITLE
Minor readme fixes

### PR DESCRIPTION
**Pull request by sten0**
Just some small things I noticed

### CODE REVIEW

**Comment by sten0** [comment]
gah, kept missing those links at the end...

**Comment by sten0** [comment]
Nate Eagleson ***@***.***> writes:

> Oh, good catches - thanks @sten0 !

You're welcome!

> Are current Debian and Ubuntu packages up-to-date? Just want to be sure we won't have similar difficulties of doing support for out-of-date versions if we remove those warnings instead of updating them.

Debian unstable/sid has 1.25.0, and stable/bookworm (aka: Debian 12) has
1.24.2.  You don't need to support 1.24.2, of course.  Debian stable is
sometimes called "Debian stale", and this well known feature is also
valued because it's mature and rigorously tested rather than cutting
edge software.  And yes, if ever a serious bug was found in php-mode
1.24.2 (ie anything that makes it nonviable) then targeted fixes are
encouraged.  It would surprise me if that happened, because every
release has at least six months of fuzzing where a package is dropped
from a release if it has a serious unfixed bug.

A method for installing the version from unstable/sid (the latest
available version) onto any release (Debian or Ubuntu) is already
documented in this README.

The paragraph I've deleted refers to unsupported releases from
2017...You can keep this paragraph if you want, but I hope you see the
irony of out-of-date documentation condemning "out-of-date versions".
The documentation will become out of date again, thus I recommend just
deleting it.  I wasn't involved with php-mode for Debian 9 (stretch),
btw, and only started looking after it in 2018.

**Comment by NateEag** [comment]
Thanks very much for the info!

I think that paragraph definitely needs updating, and appreciate you pointing it out.

I'm just not sure whether complete removal or regular updating is the best approach.

File: README.md [hunk 1]
```diff
@@ -48,8 +48,6 @@ PHP Mode is available from [package systems provided by several OSs][php-mode-pa
 
 Please consider installing the package provided in `sid` (aka: "unstable", a rolling release) in order to benefit from the latest PHP Mode features and performance improvements.  The latest version can be found here: [`elpa-php-mode`][elpa-php-mode].  Investigate "apt-pinning" to make tracking this update stream automatic.
 
-Also, the `php-elisp` package provided by [Debian 9 (stretch)][php-elisp-stretch] and [Ubuntu 18.10][php-elisp-ubuntu1810] and earlier is [extremely old][issue-430], so **PLEASE DO NOT INSTALL IT**.
-
```
Comment (NateEag) [inline README.md:52]: Poking around the readme a bit more, I see that [there was a goal of maintaining a support matrix based on Debian's support schedule](https://github.com/emacs-php/php-mode/wiki/Supported-Version#for-unixlinux-users).

Based on that, replacing this with an "unsupported OSes not supported" warning might be good? e.g.,

```suggestion
Note that there [is no support](https://github.com/emacs-php/php-mode/wiki/Supported-Version#for-unixlinux-users) for package versions installed on [end-of-life OS versions](https://wiki.debian.org/LTS).
```

Comment (sten0) [reply README.md:52]: > Poking around the readme a bit more, I see that [there was a goal of maintaining a support matrix based on Debian's support schedule](https://github.com/emacs-php/php-mode/wiki/Supported-Version#for-unixlinux-users).
> 
> Based on that, replacing this with an "unsupported OSes not supported" warning might be good? e.g.,

@zonuexe, do you want to maintain this?

@NateEag, from what I can tell that wiki thing is mostly about "what's the minimum version of Emacs that php-mode ought to support".  It doesn't make sense to me to link to it here.  Oldoldstable (currently Debian 10 "buster") has Emacs 26.1, php-mode 1.21.0-1, and the only updates buster will see are for things like security updates and ultra obscure corner case data loss bugs...and those must be backported targeted fixes.

Comment (NateEag) [reply README.md:52]: @sten0 - yup, you're right - I obviously misread that. Good point, sorry about that.

Comment (zonuexe) [reply README.md:52]: It's been a year since Debian 9 Stretch EOL, so it seems that few people are confused anymore. :+1:

**Comment by zonuexe** [comment]
Thank you all!

**Comment by sten0** [comment]
> @sten0 - yup, you're right - I obviously misread that. Good point, sorry about that.

No worries!  Thank you for taking the time to review and discuss this PR

**Comment by sten0** [comment]
> It's been a year since Debian 9 Stretch EOL, so it seems that few people are confused anymore. :+1: 

Haha, yes, this is a much more succinct explanation :)

Thanks for merging!

**Pull request merged**